rectools/models/sasrec.py

```diff
@@ -126,6 +169,11 @@ def from_dataset(cls, dataset: Dataset, n_factors: int, dropout_rate: float) ->
             raise ValueError("`item_features` in `dataset` must be `SparseFeatures` instance.")
 
         item_cat_features = item_features.get_cat_features()
+
+        if item_cat_features.values.size == 0:
```

Comment:
I don't think it's a good idea. We will have this block enabled by default. If dataset doesn't contain categorical features then this embeddings will not be created. But when these features are present they will be used.
This behaviour is the same as for all of our models that work with features (like ALS and Lightfm)